packages/babel-plugin-transform-template-literals/src/index.js

```diff
@@ -81,10 +89,15 @@ export default function(api, options) {
           this.templates.set(name, templateObject);
 
           const helperId = this.addHelper(helperName);
-          const init = t.callExpression(helperId, [
-            t.arrayExpression(strings),
-            t.arrayExpression(raws),
-          ]);
+
+          // only add raw arrayExpression if there is any difference between raws and strings
+          const init = t.callExpression(
+            helperId,
+            [
+              t.arrayExpression(strings),
+              !isStringsRawEqual ? t.arrayExpression(raws) : null,
+            ].filter(x => x),
```

Comment:
Let’s hoist out the array and push `raws` in inside an if-statement.

Reply:
Made this change